models/Github/GithubManager.js

```diff
@@ -136,6 +136,8 @@ class GithubManager {
     for (var channel of discordSubsystem.getPrimaryGuild().channels.array()) {
       if (channel.id == config.discord_coder_channel || channel.id == config.discord_coder_public_channel) {
         channel.sendEmbed(embed);
+			if(channel.id == config.discord_important_admin_channel && payload.pull_request.title.search("[admin]") != -1)
```

Comment:
nich what the fuck

Reply:
waht

Reply:
oh yeah, atom and github can't agree on which tab is good